src/js/components/summary_column_header.coffee

```diff
@@ -0,0 +1,45 @@
+React      = require 'react'
+_          = require 'lodash'
+
+ReactRouter = require 'react-router'
+Link        = React.createFactory(ReactRouter.Link)
+{div, th}   = React.DOM
+
+SummaryColumnHeader = React.createClass
+
+  clickPage: () ->
+    if @props.setPageId
+      @props.setPageId(@props.pageId)
+    else
+      console.log @props.pageId
```

Comment:
I noticed in other files you are using `console?.log?` in case it isn't available.